### ISSUE TITLE
possible to use latex syntax in research notes + wiki?

### ISSUE THREAD

**Issue by dwblair** [publiclab/plots2#125]
I.e. so that we can write $a=\frac{3}{x}$ and get:

![frac](https://cloud.githubusercontent.com/assets/639760/3667157/990caf06-11fd-11e4-9c64-6d7a8014b40d.png)

It might be neat for writing out equations w/ units in them, so that everything ends up nice and tidy and clear ...

**Comment by ananyo2012** [comment]
Oh this issue is really great. We should try to do this.

**Comment by jywarren** [comment]
It looks like expressions will have to be enclosed in tags like:

```
<math xmlns="http://www.w3.org/1998/Math/MathML">
  <mi>a</mi><msup><mi>x</mi><mn>2</mn></msup>
  <mo>+</mo> <mi>b</mi><mi>x</mi>
  <mo>+</mo> <mi>c</mi> <mo>=</mo> <mn>0</mn>
</math>
```

Interesting!

**Comment by ryzokuken** [comment]
One could write code that'd trigger such transformations in the rendering phase. 

What I mean, is a function and/or library that parses the given text for LaTeX and transforms it into such html. 

Seems like a great feature to work on, though.

**Comment by jywarren** [comment]
BTW, the most common pages where this'd be used would be on wiki and notes "show.html.erb" pages: 

https://github.com/publiclab/plots2/blob/master/app/views/notes/show.html.erb
https://github.com/publiclab/plots2/blob/master/app/views/wiki/show.html.erb

This other JavaScript lib by contrast: http://www.codecogs.com/latex/integration/htmlequations.php

accepts the format:

```
<div lang="latex">
\frac{1+sin(x)}{y}
</div>
```

Which maybe we could do using code fences? ```latex for example...

**Comment by jywarren** [comment]
I'd strongly prefer a javascript solution to this Rails-based one: https://github.com/amagical-net/rails-latex, to reduce complexity of the server-side codebase. Thanks!

**Comment by jywarren** [comment]
This was just closed in #1045!! Amazing.
